packages/interaction/src/InteractionManager.js

```diff
@@ -988,9 +1010,11 @@ export default class InteractionManager extends EventEmitter
      *  interactionEvent, displayObject and hit will be passed to the function
      * @param {boolean} [hitTest] - this indicates if the objects inside should be hit test against the point
      * @param {boolean} [interactive] - Whether the displayObject is interactive
+     * @param {boolean} [noDelayed] - Whether to process delayed events before returning. This is
```

Comment:
Documentation should include the default value when possible.
```suggestion
     * @param {boolean} [delayed=true] - Whether to process delayed events before returning. This is
```